rules/scripts/bitstreams_manifest.example.json

```diff
@@ -8,14 +8,12 @@
         "build_target": "//hw/bitstream/vivado:fpga_cw310"
       },
       "memory_map_info": {
-        "otp": {
-          "file": "otp.mmi",
-          "build_target": "//hw/bitstream/vivado:fpga_cw310"
-        },
-        "rom": {
-          "file": "rom.mmi",
-          "build_target": "//hw/bitstream/vivado:fpga_cw310"
-        }
+        "file": "memories.mmi",
+        "build_target": "//hw/bitstream/vivado:fpga_cw310",
+        "memories": [
+          "otp",
+          "rom"
+        ]
```

Comment:
This information isn't currently used in the bazel rules (not part of bitstreams_workspace.py), but I figured I'd make this info readily available in case it's useful.

The same info could be obtained from the MMI file directly, so if reviewers prefer I remove it from the manifest, I certainly can.

Reply:
Thanks, I think it is useful to know. In the future it would be interesting for us to consider creating a Bazel target that splices all memories in a single Bazel invocation.